tests/models/test_amp.py

```diff
@@ -63,7 +63,7 @@ def test_amp_single_gpu_ddp_spawn(tmpdir):
     model = BoringModel()
     # tutils.run_model_test(trainer_options, model)
     trainer.fit(model)
-
+    assert torch.is_autocast_enabled()
```

Comment:
huh, is this true? I thought it will only be enabled in the context region (training step/val step test step)

Reply:
yes you're right